product_docs/docs/pgd/3.7/harp/01_release_notes/harp2.2.2_rel_notes.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: "Version 2.2.2"
+---
+
+This is a patch release of HARP 2 that includes fixes for issues identified
+in previous versions.
+
+| Type | Description |
+| ---- |------------ |
+| Enhancement     | Upgrade dependencies to fix security vulnerabilities |
```

Comment:
I think this should be `Change` and may be we need change the description little. It was for dependabot alerts and not the direct vulnerabilities in code.

Reply:
Sure